### ISSUE TITLE
[18NewEngland] fully clear graph when needed

### ISSUE THREAD

**Issue by roseundy** [tobymao/18xx#7741]
The Quincy tile at L7 is problematic because before the first tile is laid a legal route exists just on the starting tile. After laying, a route no longer exists. When laying on that hex for the first time, clear_graph_for() needs to be called.

**Comment by philcampeau** [comment]
I don't think this is needed anymore. When the tile is laid now, rather than giving an error, it simply prompts the owner of GR to select a route, whether or not a legal route is available. This is probably better. 

If you think the graph clear is still needed, I've got a solution for that as well.

**Comment by philcampeau** [comment]
I'm going to close this for now, for the reasons I wrote in September. Feel free to reopen if you think it's still an issue.
